news/tasks.py

```diff
@@ -167,6 +167,10 @@ def et_task(func):
     @celery_app.task(base=ETTask)
     @wraps(func)
     def wrapped(*args, **kwargs):
+        start_time = kwargs.pop('_start_time', None)
```

Comment:
NIT: I'd prefer to avoid leading underscores in kwargs.
Is there an existing `start_time` kwarg that caused you to use `_start_time` as the new kwarg?